packages/now-cli/src/util/deploy/process-deployment.ts

```diff
@@ -100,11 +100,7 @@ export default async function processDeployment({
     skipAutoDetectionConfirmation,
   };
 
-  let queuedSpinner = null;
```

Comment:
The spinner logic in this file gets simplified drastically, due to the new `output.spinner()` behavior and `output.stopSpinner()` function.